addon/templates/components/lt-row.hbs

```diff
@@ -1,3 +1,7 @@
 {{#each columns as |column|}}
-  {{lt-cell column=column row=row rawValue=(get row column.valuePath) tableActions=tableActions}}
+  {{component (concat 'light-table/cells/' column.cellType)
+    column=column
```

Comment:
This is a minor thing but instead of `column=column` and `row=row` you might want to look into [positionalParams](http://emberjs.com/api/classes/Ember.Component.html#property_positionalParams)